python/guide/getting-started/to-mxnet/pytorch.md

```diff
@@ -1,15 +1,8 @@
 # MXNet for PyTorch users in 10 minutes
 
-[PyTorch](https://pytorch.org/) has quickly established itself as one of the
-most popular deep learning framework due to its easy-to-understand API and its
-completely imperative approach. But you might not be aware that MXNet includes
-the Gluon API which gives you the simplicity and flexibility of PyTorch, whilst
-allowing you to hybridize your network to leverage performance optimizations of
-the symbolic graph.
+[PyTorch](https://pytorch.org/) is a popular deep learning framework due to its easy-to-understand API and its completely imperative approach. Apache MXNet includes Gluon API which gives you the simplicity and flexibility of PyTorch and also allows you to hybridize your network to leverage performance optimizations of the symbolic graph.
```

Comment:
suggest rewriting to: "Apache MXNet includes **the** Gluon API .... Pytorch and ~~also~~ allows you to ...."